src/MusicTheory/PitchClass/Enharmonic.elm

```diff
@@ -0,0 +1,245 @@
+module MusicTheory.PitchClass.Enharmonic exposing
```

Comment:
Are you thinking that each type of note should have its own `Enharmonic` module (e.g. `PitchClass.Enharmonic`, `Pitch.Enharmonic`, `Note.Enharmonic`)? I think just one `Enharmonic` module might be enough.

Reply:
I'd prefer that if there wasn't the problem with function name clashes. Especially for pitch and pitch class there are probably a few similar functions. And I don't like prefixing very much.